lib/linkerd2-proxy-transport/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "linkerd2-proxy-transport"
+version = "0.1.0"
+authors = ["Linkerd Developers <cncf-linkerd-dev@lists.cncf.io>"]
+edition = "2018"
+publish = false
+
+[dependencies]
+bytes = "0.4"
+futures = "0.1"
+indexmap = "1.0.0"
+linkerd2-conditional = { path = "../linkerd2-conditional" }
+linkerd2-dns-name = { path = "../linkerd2-dns-name" }
+linkerd2-error = { path = "../linkerd2-error" }
+linkerd2-identity = { path = "../linkerd2-identity" }
+linkerd2-metrics = { path = "../linkerd2-metrics" }
+linkerd2-proxy-core = { path = "../linkerd2-proxy-core" }
+ring = "0.16"
+rustls = "0.16"
+tokio = "0.1.14"
+tokio-rustls = "0.10"
+tower = "0.1"
+tracing = "0.1.9"
+webpki = "0.21"
+untrusted = "0.7"
+
+[target.'cfg(target_os = "linux")'.dependencies]
+libc = "0.2"
+procinfo = "0.4.2"
+
+[dev-dependencies]
+linkerd2-identity = { path = "../linkerd2-identity", features = ["test-util"] }
+tower-util = "0.1"
+tracing-subscriber = "0.1"
```

Comment:
TIOLI: if we opted out of default features on `tracing-subscriber` we could avoid having a few dependencies in the lockfile...but maybe having nicer formatting in test output is good 🤷‍♀